pkg/querier/querier.go

```diff
@@ -229,11 +229,13 @@ func (q *Querier) Series(ctx context.Context, req *connect.Request[querierv1.Ser
 
 	responses, err := forAllIngesters(ctx, q.ingesterQuerier, func(childCtx context.Context, ic IngesterQueryClient) ([]*typesv1.Labels, error) {
 		res, err := ic.Series(childCtx, connect.NewRequest(&ingestv1.SeriesRequest{
-			Matchers: req.Msg.Matchers,
+			Matchers:   req.Msg.Matchers,
+			LabelNames: req.Msg.LabelNames,
```

Comment:
This doesn't seem to make problems with older ingesters, that do not know about the LabelNames field